src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book() {
+    }
+
+    public Book(String title, BigDecimal price) {
+        this.title = title;
+        this.price = price;
+    }
```

Comment:
When you convert `ResultSet` to `Book`, it's better to create an object using setters or constructor but not both of them. Currently, you have a constructor that sets the title and price, but the id is set via a setter. Consider adding an `id` parameter to the constructor or initializing all fields using setters.